src/generators.py

```diff
@@ -296,7 +296,7 @@ def generate_test():
         for gen in GENERATION_METHODS:
             fastqChains.extend(generate_chains(CHAINS_PER_TYPE, ALPHABET, gen, MIN_FASTQ_LENGTH, MAX_FASTQ_LENGTH))
 
-        fastq_file = open('fastq.txt', 'w')
+        fastq_file = open('fastq.fq', 'w')
```

Comment:
Hardwiring file names will almost always bite you at some point. It is much better to make them options for your program (maybe with a default if you want one).